apps/openmw/mwmechanics/actors.cpp

```diff
@@ -488,33 +552,48 @@ namespace MWMechanics
 
         float x = fFatigueReturnBase + fFatigueReturnMult * endurance;
 
-        DynamicStat<float> fatigue = stats.getFatigue();
         fatigue.setCurrent (fatigue.getCurrent() + duration * x);
         stats.setFatigue (fatigue);
     }
 
-    class ExpiryVisitor : public EffectSourceVisitor
+    struct effectStruct
```

Comment:
Pretty sure this goes against our naming conventions. Personally I would name it just 'Effect' and put it under the 'SortByRemainingTimeVisitor' namespace.